src/Documents.jl

```diff
@@ -231,6 +231,7 @@ struct User
     authors :: String
     version :: String # version string used in the version selector by default
     highlightsig::Bool  # assume leading unlabeled code blocks in docstrings to be Julia.
+    warn_outdated::Bool # insert warning popup for old versions of the documentation
```

Comment:
I think this option should be passed via the `HTML` object, since it's only relevant for the HTML backend.

Reply:
Good call. Will fix.